DittoPresenceViewer/src/main/java/ditto/live/dittopresenceviewer/DittoPresenceViewerWebView.kt

```diff
@@ -0,0 +1,69 @@
+package ditto.live.dittopresenceviewer
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.Base64
+import android.view.ViewGroup
+import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.webkit.WebViewAssetLoader
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+
+@SuppressLint("ViewConstructor")
+class DittoPresenceViewerWebView(
+    context: Context,
+    private val coroutineScope: CoroutineScope
+): WebView(context) {
+
+    private val webViewLoadingMutex = Mutex(locked = true)
+    private var updateUiJob: Job? = null
+
+    init {
+        val assetLoader = WebViewAssetLoader.Builder()
+            .addPathHandler("/assets/dist/", WebViewAssetLoader.AssetsPathHandler(context))
+            .build()
+
+        layoutParams = LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.MATCH_PARENT,
+        )
```

Comment:
Can we make this a constructor parameter? I think having a default of `MATCH_PARENT, MATCH_PARENT` is fine, but it would be nice to give folks the configurability here.